common/model/catalogue.js

```diff
@@ -4,6 +4,18 @@ export type Work = {
   ...Object,
 };
 
+// type Work = {|
```

Comment:
Why'd we comment this out?

Reply:
Didn't comment it out, I put it in there as a comment. It's essentially a duplicate of the type on line 2, which is being used everywhere, but needs filling out with what's in here. Just for a different PR I think.